src/main/java/org/project/springweb/model/Role.java

```diff
@@ -0,0 +1,31 @@
+package org.project.springweb.model;
+
+import jakarta.persistence.*;
```

Comment:
I see that you don't have checkstyle checks in your project. Let's add it and fix all of them ;)